py_zipkin/thread_local.py

```diff
@@ -43,10 +40,7 @@ def get_thread_local_span_storage():
     """
     log.warning('get_thread_local_span_storage is deprecated. See DEPRECATIONS.rst'
                 ' for details on how to migrate to using Tracer.')
-    if not hasattr(_thread_local, 'span_storage'):
-        from py_zipkin.storage import SpanStorage
-        _thread_local.span_storage = SpanStorage()
-    return _thread_local.span_storage
+    return get_default_tracer()._span_storage
```

Comment:
How about implementations for:
def get_zipkin_attrs(),
def pop_zipkin_attrs(),
push_zipkin_attrs(zipkin_attr):

They should also work with get_default_tracer()?

Reply:
Also, why not get rid of 
_thread_local = threading.local() [py_zipkin/thread_local.py].
Since only Tracer will be used and everything should be stored in _thread_local_tracer = threading.local()

Reply:
> How about implementations for: get_zipkin_attrs(), pop_zipkin_attrs(), push_zipkin_attrs(zipkin_attr)

They're also being updated since they use `ThreadLocalStack().get()`.

> Also, why not get rid of _thread_local = threading.local()

Yeah I'm already deleting it